src/activate/activate.c

```diff
@@ -216,7 +217,10 @@ static int exec_process(const char* name, char **argv, char **env, int start_fd,
                                                 return log_error_errno(r, "Failed to extend strv: %m");
                                 }
                         } else if (len != n_fds)
-                                log_warning("The number of fd names is different than number of fds: %zu vs %zu", len, n_fds);
+                                log_warning(
+                                        "The number of fd names is different than number of fds: %zu vs %zu",
+                                        len,
+                                        n_fds);
```

Comment:
Not much of an improvement. I'd prefer to exclude such changes which are not a clear improvement to avoid churn.